### PR TITLE
fix gen_sdk.sh

### PR DESCRIPTION
**Pull request by zwwhdls**
Signed-off-by: zwwhdls <zww@hdls.me>

### Ⅰ. Describe what this PR does
fix https://github.com/fluid-cloudnative/fluid/issues/1456

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: hack/sdk/gen-sdk.sh [hunk 1]
```diff
@@ -42,10 +42,10 @@ fi
 echo ">> Using ${OPENAPI_PKG}"
```
Comment (cheyang) [inline hack/sdk/gen-sdk.sh:42]: Do we still need env variables `OPENAPI_PKG`?

**Comment by cheyang** [comment]
Thanks @zwwhdls . Could you also put the swagger jar inside directory to avoid jar package downloading issue?

**Comment by zwwhdls** [comment]
@cheyang done

**Review by cheyang — APPROVED** [review]

**Comment by cheyang** [comment]
Thanks @zwwhdls

**Pull request merged**